.travis.yml

```diff
@@ -1,23 +1,18 @@
 language: node_js
 node_js:
-  - 8
-  - 10
-  - 12
+  - "8"
+  - "9"
+  - "10"
```

Comment:
can you check why this files were changed too?

Reply:
I have reverted the changes but I am looking into why this could have happened..